ReportingServicesTools/Functions/CatalogItems/Remove-RsSubscription.ps1

```diff
@@ -0,0 +1,101 @@
+﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Remove-RsSubscription
+{
+    <#
+        .SYNOPSIS
+            This function removes an subscription from the report.
+        
+        .DESCRIPTION
+            This function removes an subscription from the report.
+        
+        .PARAMETER Subscription
```

Comment:
Should you also add the SubscriptionID parameter to the list? (also in other parts of the code we typically uses Id SubscriptionId)